src/Faithlife.Utility/InvariantConvert.cs

```diff
@@ -113,6 +113,30 @@ public static string ToInvariantString(this double value)
 		/// <exception cref="FormatException">Failed to parse using invariant culture.</exception>
 		public static TimeSpan ParseTimeSpan(string text) => ThrowFormatExceptionIfNull(TryParseTimeSpan(text), text);
 
+		/// <summary>
+		/// Converts the value to a string using the invariant culture.
+		/// </summary>
+		public static string ToInvariantString(this object value)
+		{
+			switch (value)
+			{
+			case null:
+				return null;
```

Comment:
The empty string matches the behavior of `Convert.ToString`, `string.Format`, etc.

```suggestion
				return "";
```

Reply:
That's inconsistent with the other `ToInvariantString` implementations.

Reply:
Which implementations, exactly?

Reply:
I was reading the test cases. Of course the other implementations don't even deal with null because they operate on value types.